packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.extension.tsx

```diff
@@ -20,11 +25,17 @@ export default function LabOrderBasketPanelExtension() {
   const [isExpanded, setIsExpanded] = useState(orders.length > 0);
 
   const openNewLabForm = useCallback(() => {
-    launchPatientWorkspace('add-lab-order');
+    closeWorkspace('order-basket', {
+      ignoreChanges: true,
```

Comment:
Doesn't the `order-basket` workspace always ignore changes (it has nothing to change)?